pkg/boot/boot.go

```diff
@@ -63,6 +71,12 @@ func WithDryRun(dryRun bool) LoadOption {
 	}
 }
 
+func WithLoadedInfoFile(f string) LoadOption {
```

Comment:
WithLinuxConfigFile?

Reply:
please add comment for the function (i've been experimenting with some CI to add back automated comments on PRs for that, please stay tuned)

Reply:
Done.  

slightly re-worded, `WithLinuxImageCfg(...)`